codechain/config/mod.rs

```diff
@@ -101,8 +102,34 @@ impl Config {
 
         let bootstrap_addresses =
             self.network.bootstrap_addresses.iter().map(|s| SocketAddr::from_str(s).unwrap()).collect::<Vec<_>>();
-        let whitelist = self.network.whitelist.iter().map(|s| s.parse().unwrap()).collect::<Vec<_>>();
-        let blacklist = self.network.blacklist.iter().map(|s| s.parse().unwrap()).collect::<Vec<_>>();
+
+        let whitelist = self
+            .network
+            .whitelist_path
+            .as_ref()
+            .map(|path| {
+                fs::read_to_string(path)
+                    .expect("Whitelist file must exist")
```

Comment:
Not true. The given whitelist file might not exist. Handle the error case by throwing an error.